applications/Unity.GrantManager/src/Unity.GrantManager.Application/GrantApplications/ApplicationManager.cs

```diff
@@ -7,11 +7,13 @@
 using Unity.GrantManager.Identity;
 using Unity.GrantManager.Workflow;
 using Volo.Abp;
-using Volo.Abp.Domain.Services;
+using Volo.Abp.Application.Services;
 using Volo.Abp.Uow;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
 
 namespace Unity.GrantManager.Applications;
-public class ApplicationManager : DomainService, IApplicationManager
+public class ApplicationManager : ApplicationService, IApplicationManager
```

Comment:
ApplicationService ? need to move this service out of the domain if it needs to be an application service - lets discuss

Reply:
hi @AndreGAot -- yes, the reason being is so that I can use the `AuthorizationService ` class which is not available in the `DomainService`.